pkgs/development/libraries/http-parser/default.nix

```diff
@@ -21,18 +22,35 @@ stdenv.mkDerivation rec {
       sha256 = "sha256-rZZMJeow3V1fTnjadRaRa+xTq3pdhZn/eJ4xjxEDoU4=";
     })
   ];
-  makeFlags = [ "DESTDIR=" "PREFIX=$(out)" ];
+
+  makeFlags = [
+    "DESTDIR=" "PREFIX=$(out)"
+  ] ++ lib.optionals stdenv.hostPlatform.isWindows [
+    "Platform=wine"
+    "SOEXT=dll"
+    "BINEXT=.exe"
```

Comment:
Stupid question: We should use stdenv.hostPlatform.extensions.executable here, do we?

Reply:
Oh that's a very good point. Fixed now.